src/app/manager/manager-navigation.scss

```diff
@@ -0,0 +1,38 @@
+@import '../variables.scss';
```

Comment:
There are other changes to make with this component to reduce repetition that will fix this.

In the future if you are building a new component that needs the same css rather than coping that file you should reference that file in the component `styleUrls` property.